src/flag_gems/ops/attention.py

```diff
@@ -89,8 +88,7 @@ def _attn_fwd_inner(
                 qk *= 1.44269504
                 qk = qk - m_ij[:, None]
             else:
-                qk_scale *= 1.44269504
-                qk = qk * qk_scale - m_ij[:, None]
+                qk = qk * qk_scale * 1.44269504 - m_ij[:, None]
```

Comment:
Maybe better to name the constant `1.44269504` as `M_LOG2E`(as the c++ <cmath> header does), as it is the value of `log2(e)`.